### PR TITLE
feat: Bring back projects dashboard for Sentry 10

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Review by billyvg — COMMENTED** [review]
Does the projects dashboard need to become the default route again? e.g. where should `/orgId/` redirect to (currently it goes to issues).

**Comment by lynnagara** [comment]
We decided to leave the default as a redirect to issues, only bringing this back under `/projects`

File: src/sentry/static/sentry/app/views/organizationProjectsDashboard/index.jsx [hunk 1]
```diff
@@ -60,17 +65,18 @@ class Dashboard extends React.Component {
...
-            data-test-id="favorites"
-            orgId={params.orgId}
-            showBorder
-            team={null}
-            title={t('Bookmarked projects')}
-            projects={favorites}
-            access={access}
-          />
-        )}
+        {!hasSentry10 &&
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationProjectsDashboard/index.jsx]: Why is this only non-sentry10?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationProjectsDashboard/index.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationProjectsDashboard/index.jsx]: Right now there's not a guarantee that a user has access to all the bookmarked projects so linking to these could be bad. It happens in the case of org owner or open access orgs

**Pull request merged**